app/screens/ImageHandler.tsx

```diff
@@ -45,8 +45,8 @@ const ImageHandler = () => {
   const resizeImage = async ({item,Size}) => {
     try {
       const [width, height] = Size.split('x').map(Number);
-      console.log('width   :  ===  : ',width)
-      console.log('height   :  ===  : ',height)
+      // console.log('width   :  ===  : ',width)
```

Comment:
remove this consoles instead of commenting

Reply:
done